students/FredB/session04/dict_lab.py

```diff
@@ -0,0 +1,49 @@
+#Dict_lab
+#FredBallyns
+#Session04
+
+#Dictionaries 1
+dict_data = {"name": "Fred","city": "Seattle","cake": "Carrot"}
+
+print(dict_data)
+
+dict_data.pop("cake")
+
+print(dict_data)
+
+dict_data["Fruit"] = "Mango"
+
+print(dict_data)
+print(dict_data.keys())
+print(dict_data.values())
+
+print("cake" in dict_data)
+
+print("Fruit" in dict_data)
+
+#Dictionaries 2
+dict_count={}
+for i in dict_data.keys():
```

Comment:
When you want to search through the keys of a dict, you don't need to specify the .keys()

Ex: for key in dict_data: